.gitattributes

```diff
@@ -0,0 +1 @@
+package-lock.json    -diff -merge
```

Comment:
it is mixing purpose. This line is unrelated to the PR title and bug issue.

Reply:
While I can see your motivation I would suggest to not overcomplicate things. You mentioned earlier there are lots of unrelated changes in the package-lock.json and Brian was putting in a "fix" for this right here. Imho no one really cares later on if that change was done in a separate PR or not....at least not for this specific case.

Reply:
adding change that are unrelated its complicating review and future maintenance.

This is a "fix" that could potentially lead to discussion unrelated to initial request.

I care a lot about having clear sepration in PR and commits to have a maintainable software.

Reply:
Spun off the .gitattributes change to https://github.com/redhat-developer/vscode-didact/pull/602 and https://issues.redhat.com/browse/FUSETOOLS2-1161 for further discussion.